hugo-blox/blox/community/people.html

```diff
@@ -30,12 +30,12 @@ <h1 class="text-3xl font-bold mb-0">{{ . | markdownify | emojify }}</h1>
     {{ $query = sort $query $sort_by $sort_order }}
 
     {{if $query | and (gt (len $block.content.user_groups) 1) }}
-      <div class="col-span-full mb-6">
-        <h2 class="text-2xl font-semibold mb-4">{{ . | markdownify }}</h2>
+      <div class="col-span-full my-4">
+        <h2 class="text-2xl font-semibold">{{ . | markdownify }}</h2>
       </div>
     {{end}}
 
-    <div class="grid grid-cols-2 md:grid-cols-5 lg:grid-cols-6 gap-3">
+    <div class="flex flex-row flex-wrap" style="justify-content: space-evenly">
```

Comment:
好みの問題なのですが、`space-evenly` だと、絶妙な画面サイズのときに最後の段がまばらになってしまうのが少し気になりました。

<img width="989" alt="Screenshot 2024-11-26 at 14 03 43" src="https://github.com/user-attachments/assets/e977c01c-3343-4f4f-b2be-73011b193d77">

`justify-center` だとどうでしょう?

```suggestion
    <div class="flex flex-row flex-wrap justify-center">
```

<img width="992" alt="Screenshot 2024-11-26 at 14 03 59" src="https://github.com/user-attachments/assets/fc20110c-0426-47f6-9539-abd5979cb805">


(余談: 何故かhugo-bloxで `justify-center`は動くけど `justify-evenly` が動かない...(だからinline CSS使ったんだと思うのですが)、なんでなんでしょうね〜🤔 )